packages/js/src/Modules/Verto/BrowserSession.ts

```diff
@@ -201,6 +201,28 @@ export default abstract class BrowserSession extends BaseSession {
 
   /**
    * Return the device list supported by the browser
+   *
+   * ## Examples
+   *
+   * Using async/await:
+   *
+   * ```js
+   * async function() {
+   *   const client = new TelnyxRTC(options);
+   *
+   *   let result = await client.getDevices();
+   *
+   *   console.log(result);
+   * }
+   * ```
```

Comment:
Feels better to have the code without spaces
```suggestion
   * ```js
   * async function() {
   *   const client = new TelnyxRTC(options);
   *   let result = await client.getDevices();
   *   console.log(result);
   * }
   * ```
```